dev/run-tests.py

```diff
@@ -128,7 +128,7 @@ def determine_tags_to_exclude(changed_modules):
     tags = []
     for m in modules.all_modules:
         if m not in changed_modules:
-            tags += m.test_tags
+            tags += m.tags
```

Comment:
Paging @vanzin because of https://github.com/pravingadakh/spark/commit/94fc57afdf8ac6be35f13956232b6cf58857d047 -- I don't know the history of this bit